distribution/changelog.txt

```diff
@@ -12,6 +13,8 @@
 - Fix: [#25163] Some of the Junior Roller Coaster flat to steep track wooden support clearance heights are different to RCT1.
 - Fix: [#25173] Desync when placing a park entrance in multiplayer.
 - Fix: [#25179] The LIM Launched Roller Coaster inline twists have incorrect wooden support clearance heights (original bug).
+- Fix: [#25192] Fix faulty alphabetical sorting in certain cases when changing filter key in the Ride List Window.
+- Fix: [#25192] Fix flip-flapping of entries in the Ride List Window under certain cases.
```

Comment:
Unless there are actual issues for this, I think the first of these two will do.